catalogue/webapp/components/IIIFViewer/IIIFViewer.tsx

```diff
@@ -329,15 +324,17 @@ const IIIFViewer: FunctionComponent<IIIFViewerProps> = ({
   useEffect(() => {
     const matchingManifest =
       parentManifest &&
-      parentManifest.manifests &&
-      parentManifest.manifests.find((childManifest: CollectionManifest) => {
+      parentManifest.items &&
+      parentManifest.items.find((canvas: Canvas) => {
         return !transformedManifest
           ? false
-          : childManifest['@id'] === transformedManifest.id;
+          : canvas.id === transformedManifest.id;
       });
 
-    matchingManifest && setCurrentManifestLabel(matchingManifest.label);
-  });
+    // FIXME: this en[1] feels like a bad idea
```

Comment:
What do you think alternatives could be, if not using en[1] as per comment

Reply:
Can't see any tbh